packages/block-editor/src/hooks/background.js

```diff
@@ -302,18 +340,150 @@ function BackgroundImagePanelItem( { clientId, setAttributes } ) {
 	);
 }
 
+function backgroundSizeHelpText( value ) {
+	if ( value === 'cover' || value === undefined ) {
+		return __( 'Stretch image to cover the block.' );
+	}
+	if ( value === 'contain' ) {
+		return __( 'Image is contained within the block.' );
+	}
+	return __( 'Repeat the image, and set a fixed size.' );
```

Comment:
I don't know if this matters, but should we be clear that the value affects the `width` only? (height is `auto`)

I believe I'm overthinking it here.

Reply:
I forgot to add to my comments, I know this is WIP so feel free to ignore all my "observations" 😄

Reply:
Oh, good point! I've been using squarish images in my testing so didn't notice. We can change the help text to say `fixed width` instead 👍

Reply:
Updated the help text in https://github.com/WordPress/gutenberg/pull/57005/commits/53e38e343c415bd325681450f646a6891b823bae